data/faq.md

```diff
@@ -2,9 +2,9 @@
 
 ## **Q. When and where is EuroPython 2022 taking place?**
 
-A. After 2 years of being fully remote, EuroPython 2022 will be held on**11th-17th July**and it will be both in person and virtual.
+A. After 2 years of being fully remote, EuroPython 2022 will be held on **11th-17th July** and it will be both in person and virtual.
 
-The in-person conference will be held at [**The** **Convention Centre Dublin (The CCD**)](https://www.theccd.ie/)** in Dublin**,**Ireland**.
+The in-person conference will be held at [**The** **Convention Centre Dublin (The CCD**)](https://www.theccd.ie/) **in Dublin,** **Ireland**.
```

Comment:
I think this could be simplified to just using `**(All the content)**` instead of splitting it to multiple parts

Reply:
good catch, fixed it..